src/styles/AdminDashboardHome/index.less

```diff
@@ -0,0 +1,79 @@
+h1,
+h2 {
+  color: white;
+}
+
+.AdminDash {
+  background-color: #006c72;
+  margin-bottom: 10px;
+}
+
+.AdminDash h1 {
+  padding-top: 0.5%;
+  padding-left: 1%;
+  padding-bottom: 1%;
+  font-family: Roboto;
+  font-size: 30px;
+}
+
+.container {
+  display: flex;
+  flex-flow: row nowrap;
+  background-color: #263e47;
+  padding-top: 0.5%;
+  padding-bottom: 0.05%;
+  margin-top: 5px;
+  font-size: 22px;
+  padding-left: 1%;
+  width: 50%;
+}
+
+.courses {
+  margin-right: 10px;
+}
```

Comment:
There is a very slight responsiveness problem with both the courses and admin heading. The "See All" link does not stay within the background as the screen gets smaller, and becomes difficult to see

Reply:
We looked into this and have fixed it to the best of our ability.